src/sparseml/pytorch/torchvision/train.py

```diff
@@ -408,7 +409,14 @@ def collate_fn(batch):
     if args.distributed and args.sync_bn:
         model = torch.nn.SyncBatchNorm.convert_sync_batchnorm(model)
 
-    criterion = nn.CrossEntropyLoss(label_smoothing=args.label_smoothing)
+    min_torch_version_with_label_smoothing = version.parse("1.10")
+    label_smoothing_supported = (
+        version.parse(torch.__version__) >= min_torch_version_with_label_smoothing
+    )
+    if label_smoothing_supported:
+        criterion = nn.CrossEntropyLoss(label_smoothing=args.label_smoothing)
+    else:
+        criterion = nn.CrossEntropyLoss()
```

Comment:
Let's add a `raise ValueError` here if args.label_smoothing > 0 so we aren't silently not doing something